lib/text-class/test/unit/Data/Text/ClassSpec.hs

```diff
@@ -48,6 +58,11 @@ spec = do
             toText @Text "patate" === "patate"
         it "fromText . toText === pure" $
             property $ \(t :: Text) -> (fromText . toText) t === pure t
+        it "fromText ~ fromTextMaybe" $
```

Comment:
Perhaps this should be in a separate category? (It appears to be a test of `fromTextMaybe`, but this section is about the `FromText Text` instance.)

Reply:
by symmetrical argument we can say this is also the test for `fromText` and then it fits here ;-)

Reply:
I agree with both statements, fixed.